herbert_core/classes/MPIFramework/@ClusterMPI/private/check_mpi_mpiexec_can_be_enabled_.m

```diff
@@ -1,18 +1,20 @@
 function check_mpi_mpiexec_can_be_enabled_(obj)
 
-ok = true;
-mess = [];
-if isempty('cpp_communicator')
-    ok = false;
+
+if isempty(which('cpp_communicator'))
     mess = 'Can not find cpp_communicator mex file on Matlab search path';
+    error('PARALLEL_CONFIG:not_available',mess);
+end
+try
+    ver = cpp_communicator();
```

Comment:
what is `ver` used for here?

Reply:
ver is not used, but mex file interface, returning ver needs this kind of call -- one arg-out and no arg-in, so have to be here.

Reply:
Doesn't Matlab support dummy argument/return values (`~`)? Having unused variables declared makes the code more confusing

Reply:
Matlab supports, but I am not sure how mex would recognize it -- as an argument or no argument. I suspect as no argument, because it should not generate output value then. 
needs more investigations...
Check within custom mex return version in case of one output and no input arguments and I do not think its worth chaning this.